### PR TITLE
Procedural terrain 1D

### PR DESCRIPTION
**Pull request by svera**
Demo of a 1D terrain generator using Perlin noise.

### CODE REVIEW

File: examples/community/procedural-terrain-1d/README.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ Press *space* to generate a random terrain.
 Uses [Go-Perlin](https://github.com/aquilax/go-perlin).
 
 Texture by [hh316](https://hhh316.deviantart.com/art/Seamless-stone-cliff-face-mountain-texture-377076626).
+
+![](result.png)
```
Comment (mewmew) [inline examples/community/procedural-terrain-1d/README.md]: Consider adding a title for the image,. such that if it fails to load, or if someone with visual impairment visits the site, there exists an alternative text for presenting what the image is about.

E.g.

```
![Title for image](result.png)
```

Also, is result.png missing from the PR?

File: examples/community/procedural-terrain-1d/main.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	"math/rand"
+	"os"
+	"time"
+
+	_ "image/jpeg"
+
+	perlin "github.com/aquilax/go-perlin"
+	"github.com/faiface/pixel"
+	"github.com/faiface/pixel/pixelgl"
+	"github.com/svera/pixel/imdraw"
```
Comment (peterhellberg) [inline examples/community/procedural-terrain-1d/main.go]: The wrong imdraw package is imported here.

**Comment by faiface** [comment]
You fellows highlighted all the problems in this PR, that I'm aware of. Once those get resolved, I'm ready to merge this. Thank you @svera for yet another cool community example!

**Comment by svera** [comment]
@mewmew @peterhellberg @faiface thank you for the comments, I'll address these ASAP and update the PR accordingly.

**Comment by svera** [comment]
@faiface changes done.

**Comment by faiface** [comment]
Very good!

**Pull request merged**